src/app/scenes/Category.tsx

```diff
@@ -1,56 +1,59 @@
 import React from 'react';
```

Comment:
```suggestion
import React, { useEffect } from 'react';
```

useEffect 도 다른 hook 들과 일관되게 사용하는게 좋을 것 같아요.

Reply:
서점과 셀렉트에서 두가지 방법이 모두 혼용되고 있는데요
하나로 정리하는게 좋을 것 같아요.
셀렉트는 다른 hook 들과 일관되게 사용하는 것으로 할게요.
점진적으로 수정해나가겠습니다.